packages/charting/src/components/DonutChart/Arc/Arc.types.ts

```diff
@@ -0,0 +1,56 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { IDataPoint } from '../DonutChart.types';
+
+export interface IArcProps {
+  /**
+   * Data to render in the Arc.
+   */
+  data?: IArc;
+
+  /**
+   * innerRadius of the Arc.
+   */
+  innerRadius?: number;
+  /**
+   * outerRadius of the Arc.
+   */
+  outerRadius?: number;
+  /**
+   * Color for the Arc.
+   */
+  color: string;
+}
+
+export interface IArc {
```

Comment:
This name is a little confusing as I thought it would be implemented by the component. I wonder if `IArcData` would be more clear? Also, do you really mean for ALL of this data to be optional? Should IArcData have a minimal data set?